python/lsst/pipe/tasks/scaleVariance.py

```diff
@@ -26,6 +26,8 @@
 from lsst.pipe.base import Task
 from lsst.meas.algorithms import SubtractBackgroundTask
 
+__all__ = ["ScaleVarianceConfig", "ScaleVarianceTask"]
+
```

Comment:
This is part of the "Add docs" commit, right.

Reply:
Automodapi may work without this, but I think it's better to have it explicitly written for it.